admin/class-collapsible-presenter.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Collapsible_Presenter.
+ */
+class WPSEO_Collapsible_Presenter extends WPSEO_Paper_Presenter {
+
+	/**
+	 * WPSEO_presenter_collapsible constructor.
+	 *
+	 * @param string $title     The title of the collapsible.
+	 * @param string $view_file Optional. The path to the view file. Use the content setting if you do not wish to use
+	 *                          a view file.
+	 * @param array  $settings  Optional. Settings for the collapsible.
+	 */
+	public function __construct( $title, $view_file = null, array $settings = [] ) {
+		$defaults = [
+			'paper_id'                 => null,
+			'paper_id_prefix'          => 'wpseo-',
+			'collapsible'              => false,
+			'collapsible_header_class' => '',
+			'expanded'                 => false,
+			'help_button'              => '',
+			'title_after'              => '',
+			'class'                    => '',
+			'content'                  => '',
+			'view_data'                => [],
+		];
+
+		$settings = wp_parse_args( $settings, $defaults );
+		$title     = $title;
+		$view_file = $view_file;
+
+		parent::__construct( $title, $view_file, $settings );
```

Comment:
I'm not sure calling the parent constructor this way is a good pattern. Not my area of expertise though so I'd like this to be checked by a senior PHP dev.

Reply:
This has been accepted by Jip